src/hotspot/share/runtime/continuationFreezeThaw.cpp

```diff
@@ -1014,7 +1021,7 @@ void FreezeBase::patch(const frame& f, frame& hf, const frame& caller, bool is_b
 
   if (f.is_interpreted_frame()) {
     assert(hf.is_heap_frame(), "should be");
-    ContinuationHelper::InterpretedFrame::patch_sender_sp(hf, caller.unextended_sp());
+    ContinuationHelper::InterpretedFrame::patch_sender_sp(hf, caller);
```

Comment:
Thanks for removing `unextended_sp`, here. It's not generic enough for shared code.